src/main/frontend/db/query_react.cljs

```diff
@@ -105,7 +105,7 @@
           inputs (cond-> resolved-inputs
                          rules
                          (conj rules))
-          k [:custom (or (:query-string query') query')]]
+          k [:custom (or (:query-string query') (assoc query' :inputs inputs))]]
```

Comment:
is it a good idea to add a test to avoid regression?

Reply:
Of course. I tried to draft a test in https://github.com/logseq/logseq/commit/9450e4889ccd48840d8685ce25767b35c4351f95, but it does not trigger the issue as expected. Did you know what's wrong there?